src/sugar3/graphics/toolbutton.py

```diff
@@ -101,13 +125,19 @@ def set_tooltip(self, tooltip):
         Gtk.ToolButton.set_label(self, tooltip)
 
     def get_tooltip(self):
+        '''
+        Returns the above tooltip string.
+        '''
         return self._tooltip
 
     tooltip = GObject.property(type=str, setter=set_tooltip,
                                getter=get_tooltip)
 
     def get_hide_tooltip_on_click(self):
         return self._hide_tooltip_on_click
+        '''
+        Hides tooltip when clicked
+        '''
```

Comment:
Documentation should be after the `def` and before the first executable line `return`.